evalme/image/brush.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
```

Comment:
```suggestion
# This code was copy-pasted from `label_studio_converter.brush.decode_rle` and should be kept in sync with that function.
# It can be removed if `evalme` is merged into SDK.
```